web-common/src/features/dashboards/big-number/MeasureBigNumber.svelte

```diff
@@ -42,7 +42,11 @@
   $: measureIsPercentage = formatPresetEnum === FormatPreset.PERCENTAGE;
 </script>
 
-<div class="flex flex-col pl-1 {withTimeseries ? 'mt-2' : 'justify-between'}">
+<button
```

Comment:
This `button` doesn't have an `on:click`

Reply:
Yes, had `on:click` before but removed it. Have added these changes for the upcoming TDD feature.